BitFlow/Eval/TorchEval.py

```diff
@@ -21,8 +21,25 @@ def eval_Round(self, a, prec, node: DagNode):
         scale = 2.0**prec
         return IntRound(a * scale)/scale
 
+    def eval_Relu(self, a, node: DagNode):
```

Comment:
Unfortunately this will not quite work if you are passing in a torch array. Can you do a test for this case? I suspect there is already a torch.relu function somewhere which you can use.